skyportal/handlers/api/instrument.py

```diff
@@ -975,3 +965,48 @@ def add_tiles(
     finally:
         Session.remove()
         return field_ids
+
+
+class InstrumentFieldHandler(BaseHandler):
+    @permissions(['Delete instrument'])
```

Comment:
Is this the correct permissions for this action?

Reply:
I think so. We want people with that level to do this.